CHANGELOG.adoc

```diff
@@ -23,6 +23,19 @@ always writes them under the .dfx/ directory.
 
 == DFX
 
+=== feat!: move all the flags to the end
+
+Command flags have been moved to a more traditional location; they are no longer positioned per subcommand, but instead are all positioned after the final subcommand. In prior versions, a command might look like:
+[source, bash]
+----
+dfx --identity alice canister --network ic --wallet "$WALLET" create --all
+----
+This command should now read:
+[source, bash]
+----
+dfx canister create --all --network ic --wallet "$WALLET" --identity alice
+----
+
```

Comment:
This should go in the `UNRELEASED` section above

Reply:
Good catch!